packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -165,6 +198,11 @@ export class GuxAvatar {
     return `${this.name}`;
   }
 
+  private onAvatarClick(): void {
+    if (!this.changePhoto) return;
+    this.edit.emit();
+  }
```

Comment:
I think this could be simplified to 

`if(this.changePhoto) {
this.edit.emit();
}
`

to avoid the negation.
`